### PR TITLE
Sass refactor

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description

This PR...
- adds media queries for reactivity
- adds slight adjustments to card styling
- adds recipe-display pop-up section to html
- makes sure the link looks uniform 

## Affected areas of application

- styles.css
- styles.css.map
- styles.scss
- pantry.html
- index.htm;
- scripts.js

## How Has This Been Tested?

- Chrome browser

## Screenshots
Screen Shot 2020-09-13 at 2 02 29 PM
Screen Shot 2020-09-13 at 2 02 13 PM
Screen Shot 2020-09-13 at 2 01 56 PM

### CODE REVIEW

File: src/css/styles.scss [hunk 1]
```diff
@@ -274,3 +340,15 @@ main {
   background-image: url(https://image.flaticon.com/icons/svg/148/148841.svg);
   background-repeat: no-repeat;
 }
+
+@media (min-width: 800px) {
```
Comment (nicolegooden) [inline src/css/styles.scss:344]: Thanks so much for adding these media queries in! From the screenshot, they look awesome.

File: src/pantry.html [hunk 2]
```diff
@@ -18,7 +18,7 @@ <h1 class="open-menu-title x">Menu</h1>
     </header>
     <section class="open-menu-selections x">
       <article class="menu-my-pantry menu-selection x">
-        <h3 class="menu-my-pantry-title">My Pantry</h2>
+        <h3 class="menu-all-recipe-title">All Recipes</h2>
```
Comment (nicolegooden) [inline src/pantry.html:21]: is this the 'back to home' version of what we were talking about earlier? does this take the user back to the main page?

File: src/index.html [hunk 3]
```diff
@@ -15,17 +15,17 @@
...
     <section class="open-menu-selections x">
       <article class="menu-my-pantry menu-selection x">
         <h2 class="menu-my-pantry-title"><a href="./pantry.html">My Pantry</a></h2>
       </article>
-      <article class="menu-my-upcoming-recipes menu-selection hidden">
+      <article class="menu-my-upcoming-recipes menu-selection">
         <h3 class="menu-my-upcoming-recipes-title">My Upcoming Recipes</h2>
       </article>
-      <article class="menu-favorites menu-selection hidden">
+      <article class="menu-favorites menu-selection">
```
Comment (nicolegooden) [inline src/index.html]: Let's make sure we add that `.hidden` class back in so that we have what we need on initial page load.

**Review by nicolegooden — APPROVED** [review]

File: src/css/styles.scss [hunk 4]
```diff
@@ -79,6 +99,17 @@ h1 {
     color: $secondary-color;
   }
 
+  a {
+    text-decoration: none;
+    font-size: 2rem;
+  }
+  a:visited {
```
Comment (taylorjohnson141) [inline src/css/styles.scss:106]: I love the use of variables here!

File: src/css/styles.scss (hunk 1 above)
Comment (taylorjohnson141) [reply src/css/styles.scss:344]: I agree your spacing looks great!

**Review by taylorjohnson141 — APPROVED** [review]
Looks Great!

**Pull request merged**